.github/workflows/build.yml

```diff
@@ -83,6 +83,14 @@ jobs:
         run: python .github/scripts/checkstyle.py
         env:
           GITHUB_WORKSPACE: ${{ github.workspace }}
+      - name: Update Floating 'cached' Tag on Successful Trunk Build
+        if: github.ref == 'refs/heads/trunk' && success()
+        run: |
+          git fetch --tags
```

Comment:
I think you can fetch a single tag here to save a bit of time.

Reply:
Sure!